br-webui/views/system.liquid

```diff
@@ -240,20 +240,181 @@ socket.on('cpu stats', function(data) {
 	document.getElementById('ramTotal').innerHTML = data.ram_total.toFixed(3) + ' MB';
 	document.getElementById('cpuStatus').innerHTML = data.cpu_status;
 });
+</script>
 
-socket.on('getmav', function(data) {
-	document.getElementById('mav').innerHTML = data.mav;
-	document.getElementById('vid').innerHTML = data.vid;
-	document.getElementById('webterm').innerHTML = data.webterm;
-	document.getElementById('aud').innerHTML = data.aud;
-	document.getElementById('web').innerHTML = data.web;
-	document.getElementById('filemanager').innerHTML = data.filemanager;
-	document.getElementById('router').innerHTML = data.router;
-	document.getElementById('nmearx').innerHTML = data.nmearx;
-	document.getElementById('driver').innerHTML = data.driver;
-	document.getElementById('pingproxy').innerHTML = data.pingproxy;
-});
 
+
+<script>
+    var companionDevices =
```

Comment:
Is it possible to share this variable with the test ?
Or a function that returns that to the test class to avoid redundant code.

Reply:
Probably. This is the idea. I don't know how to do it right now, I need to learn how. https://github.com/bluerobotics/companion/pull/210#discussion_r241463766

Reply:
Ok, we should create a issue for that after merging this.

Reply:
https://github.com/bluerobotics/companion/issues/214